test/pylib/scylla_cluster_manager.py

```
@@ -0,0 +1,1487 @@
+            if not task.done():
+                self.logger.info("wait for task:%s, op:%s", task, descr)
+                try:
+                    await asyncio.wait_for(task, timeout=120)
+                except asyncio.TimeoutError:
+                    self.break_manager(f"error on waiting coro {task.get_name()}", self.current_test_case_full_name)
+                    break
+                except Exception:
+                    # The operation failed after its caller went away.  It was
+                    # already logged by op_finished; completing is all the
+                    # drain needs.
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)